src/ngraph/runtime/cpu/pass/cpu_assignment.cpp

```diff
@@ -483,6 +484,30 @@ namespace ngraph
                     }
                 }
 
+                template <>
+                void CPUAssignment::ASSIGN_DECL(ngraph::op::ReplaceSlice)
+                {
+                    auto replace_slice = static_cast<op::ReplaceSlice*>(node);
+
+                    //auto arg0_shape = node->get_input_shape(0);
+                    //auto arg0_rank = arg0_shape.size();
+                    //auto result_shape = node->get_output_shape(0);
+
+                    //TO confirm : not checking ranks as we support ranks 0/1/2/4
+                    if (node->get_input_element_type(0) == element::f32)
```

Comment:
I think we can support any supported element type.

Reply:
I think so too. 
we would need to check if the type for arg0 and arg1 are of same type